specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,232 @@
+
+We propose an SBT standard to model protocols described above.
+
+## Specification
+
+Soulbound tokens need to be recoverable in case a user's private key is compromised (due to extortion, loss, etc). This becomes especially important for proof-of-human stamps and other NFTs that can only be issued once per user.
+
+Two safeguards against misuse of recovery are contemplated. 1) Users cannot recover an SBT by themselves. The issuer, a DAO or a smart contract (eg: multisig) dedicated to manage the recovery should be assigned. 2) Whenever a recovery is triggered then the wallet from which the NFT was recovered gets blacklisted (indicator that the account identity is burned). Recovering one SBT triggers a blacklist that should apply for SBTs that share the same address.
+
+Soulbound tokens with expire date SHOULD have an option to be renewable. Examples include mandatory renewal with some frequency to check that the owner is still alive, or renew membership to a DAO that uses SBTs as membership gating.
+
+### Smart contract interface
+
+The Soulbound Token interface is a subset of NFT, hence the interface follows the [NEP-171](https://github.com/near/NEPs/blob/master/neps/nep-0171.md).
+
+```rust
+/// TokenMetadata defines attributes for each SBT token.
+pub struct TokenMetadata {
+    pub title: Option<String>, // ex. "fist bump with Robert"
+    pub description: Option<String>, // free-form description
+    pub media: Option<String>, // URL to associated media, preferably to decentralized, content-addressed storage
+    pub media_hash: Option<Base64VecU8>, // Base64-encoded sha256 hash of content referenced by the `media` field. Required if `media` is included.
+    pub copies: Option<u64>, // number of copies of this set of metadata in existence when token was minted.
+    pub issued_at: Option<u64>, // When token was issued or minted, Unix epoch in milliseconds
+    pub expires_at: Option<u64>, // When token expires, Unix epoch in milliseconds
+    pub starts_at: Option<u64>, // When token starts being valid, Unix epoch in milliseconds
+    pub updated_at: Option<u64>, // When token was last updated, Unix epoch in milliseconds
+    pub extra: Option<String>, // anything extra the SBT wants to store on-chain. Can be stringified JSON.
+    pub reference: Option<String>, // URL to an off-chain JSON file with more info.
+    pub reference_hash: Option<Base64VecU8>, // Base64-encoded sha256 hash of JSON from reference field. Required if `reference` is included.
+}
+
+
+trait SBT {
+    /**********
+    * QUERIES
+    **********/
+
+    /// get the information about specific token ID
+    fn sbt(&self, token_id: TokenId) -> Option<Token>;
```

Comment:
may consider `nft` prefix where relevant to make the functions compatible with existing tools. eg : `nft_tokens_for_owner` instead of `sbt_tokens_for_owner`.

Reply:
Collision with the existing tooling will create more harm than help here as we already discussed in #359. The only way I can see that being resolved is by the NEP which will help to identify the supported extensions, read more in #129, #154, and #275.

Reply:
Actually, I was trying to refer to #351, which could be then used as a requirement for SBT NEP to unlock `nft_` prefixes by saying that SBT = NEP-171 + NEP-330 + NEP-XXX (that just defines that `nft_transfer` is not available) - this way the tooling can reasonably handle the deviation from NEP-171 through introspection.

Reply:
I was bringing up interface registration already in 2020, then created #154 ;)

Reply:
SBT is a subset of NFT, not an extension. Adding #330 makes sense.
It will make sense to use subset of it's interface methods.